### PR TITLE
ADSEC-2019 create sample files for using the api with excel

### PR DESCRIPTION
**Pull request by AnishSanghai123**
[ADSEC-2019]

[ADSEC-2019]: https://ovearup.atlassian.net/browse/ADSEC-2019?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: AdSec/Python/ApiToExcel.py
Comment (renefritze) [inline AdSec/Python/ApiToExcel.py:1]: I'd like to suggest setting up a `pre-commit` config to automatically format and lint Python code with `ruff`. That would take care of some improvements I'd suggest to the example too.

Comment (renefritze) [reply AdSec/Python/ApiToExcel.py:1]: https://github.com/arup-group/oasys-api-samples/pull/23

Comment (AnishSanghai123) [reply AdSec/Python/ApiToExcel.py:1]: Hi @renefritze ,
We have created another ticket :- https://ovearup.atlassian.net/browse/ADSEC-2030 to set up the pre-commit with ruff.

Comment (AnishSanghai123) [reply AdSec/Python/ApiToExcel.py:1]: Hey @renefritze ,
Thanks for looking into this 
Could you please resolve the conversation?

**Pull request closed without merging**